src/snapred/ui/presenter/LogTablePresenter.py

```diff
@@ -75,7 +75,7 @@ def updateButton(self, text):
     def handle_button_clicked(self):
         self.view.button.setEnabled(False)
 
-        request = SNAPRequest(path="/reduction", payload=RunConfig(runNumber="48741").json())
+        request = SNAPRequest(path="/reduction", payload=RunConfig(runNumber="45158").json())
```

Comment:
Is there a reason for the changed runNumber?

Reply:
I was running some manual testing!

Reply:
I will revert the changes!